### PR TITLE
Added timestamp to wzrk.id

### PR DESCRIPTION
**Pull request by PraveenCTzen**
JIRA Issue: [WEB-2664](https://wizrocket.atlassian.net/browse/WEB-2664)
​

**Background**
In this PR, we address an issue encountered when multiple web push notifications with distinct deeplinks were sent from the same campaign. Previously, clicking on an older notification redirected users to the latest deeplink instead of the corresponding one, causing inconsistencies.

**Implementation**
This fix ensures that each notification correctly redirects to its respective deeplink.
​

**Testing Strategy**
Successfully tested the solution using a campaign that sends out varied push notifications, each with a distinct deeplink tailored to different profile properties. Verified that all push notifications redirect correctly. Assistance is now required to replicate and test these scenarios with API campaigns.

**Impact Areas**
The impacted areas primarily include scenarios involving web push notifications with deeplinks. The issue arises when a campaign includes multiple push notifications, each with its own unique deeplink. This setup may lead to inconsistent redirection.

[WEB-2664]: https://wizrocket.atlassian.net/browse/WEB-2664?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by AmeeParsania** [comment]
@PraveenCTzen Please update the description with the below template - 

JIRA Issue: JIRA_TICKET_ID
​Background
​Implementation
​Testing Strategy
Impact Areas

File: sw_webpush.js [hunk 1]
```diff
@@ -37,13 +37,19 @@ self.addEventListener('activate', function (event) {
...
+  var currentSecond = now.getSeconds()
+  const formattedTime = currentHour + '_' + currentMinute + '_' + currentSecond
   console.log('Push event: ', event)
   // get all the notification data
   var notificationData = JSON.parse(event.data.text())
   var title = notificationData.title
   var notificationOptions = notificationData.notificationOptions
   var data = notificationOptions.data
   var key
+  data.wzrk_id = data.wzrk_id + formattedTime
```
Comment (AmeeParsania) [inline sw_webpush.js:52]: can we do  data.wzrk_id += new Date().getTime() ??

Also what;s the type of data.wzrk_id ?

Comment (AmeeParsania) [reply sw_webpush.js:52]: you can prefix the timestamp with _ if you wish to

Comment (PraveenCTzen) [reply sw_webpush.js:52]: Sure, I'm trying this "data.wzrk_id += new Date().getTime() "
Also the type of "data.wzrk_id " is string

**Comment by AmeeParsania** [comment]
@PraveenCTzen Also test if the notification viewed and clicked events are getting raised with the right information.

File: sw_webpush.js [hunk 2]
```diff
@@ -37,13 +37,13 @@ self.addEventListener('activate', function (event) {
 })
 
 self.addEventListener('push', function (event) {
-  console.log('Push event: ', event)
```
Comment (AmeeParsania) [inline sw_webpush.js:40]: undo this change.

Comment (PraveenCTzen) [reply sw_webpush.js]: Done

**Comment by PraveenCTzen** [comment]
> @PraveenCTzen Also test if the notification viewed and clicked events are getting raised with the right information.

Yes notification viewed and clicked events are getting raised properly

**Review by AmeeParsania — APPROVED** [review]

**Review by kkyusuftk — APPROVED** [review]
